src/main/kotlin/uk/gov/justice/digital/hmpps/makerecalldecisionapi/service/RiskService.kt

```diff
@@ -81,11 +81,18 @@ internal class RiskService(
         circumstancesIncreaseRisk = circumstancesIncreaseRisk,
         factorsToReduceRisk = factorsToReduceRisk,
         whenRiskHighest = whenRiskHighest,
-        activeRecommendation = recommendationDetails
+        activeRecommendation = recommendationDetails,
+        assessmentStatus = getAssessmentStatus(crn)
       )
     }
   }
 
+  suspend fun getAssessmentStatus(crn: String): String {
+    val assessmentsResponse = fetchAssessments(crn)
+    val latestAssessment = assessmentsResponse.assessments?.maxByOrNull { LocalDateTime.parse(it.initiationDate).toLocalDate() }
+    return if (latestAssessment?.superStatus == "COMPLETE") "COMPLETE" else "INCOMPLETE"
```

Comment:
Maybe it's fine... but could be a good idea to make this return an Enum that can be reused wherever you want to check the assessment status? Up to you.

Reply:
Done